test/Microsoft.AspNetCore.Mvc.Localization.Test/Internal/MvcLocalizationServiceCollectionExtensionsTest.cs

```diff
@@ -31,6 +31,37 @@ public void AddLocalizationServices_AddsNeededServices()
 
             // Assert
             Assert.Collection(collection,
```

Comment:
Could we change these tests to use the pattern from https://github.com/aspnet/Mvc/blob/dev/test/Microsoft.AspNetCore.Mvc.Core.Test/DependencyInjection/MvcCoreServiceCollectionExtensionsTest.cs? We really don't want to verify what services dependencies add, just the ones we care about.